modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/resource/PersonCollectionResource.java

```diff
@@ -321,6 +330,14 @@ private UserWrapper _updateUser(
 
 		Date birthdayDate = user.getBirthday();
 
+		Calendar calendar = Calendar.getInstance();
+
+		calendar.setTime(birthdayDate);
+
+		int birthdayDay = calendar.get(Calendar.DATE);
```

Comment:
How is sorted in other resources? I don't know if you have to sort it alphabetically or by use.

Reply:
https://github.com/liferay/liferay-portal/blob/79fd15fc19427c804c90143f37b96a3466be6c36/modules/apps/headless-apio/person/person-apio-impl/src/main/java/com/liferay/person/apio/internal/architect/form/PersonCreatorForm.java#L201